extra/exe-builder/Program.cs

```diff
@@ -49,8 +57,9 @@ public class UptimeKumaApplicationContext : ApplicationContext
 
         private RegistryKey registryKey = Registry.CurrentUser.OpenSubKey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Run", true);
 
+        private readonly bool intranetOnly = false;
```

Comment:
```suggestion
        private readonly bool intranetOnly;
```
I am not a csharp expert (have never written a line.)
Initialising both with false when we are overriding this in the inalisation seems weird. What is the rationale behind this?

Reply:
True, the default value is also false, this was just for readability.
I'll apply this change + the others today